api/controllers/question.js

```diff
@@ -23,23 +23,62 @@ questionRouter.post('/', tokenExtractor, userExtractor, async (request, response
     return response.status(401).end()
   }
 
-  const savedQuestion = await question.save()
-  return response.status(201).json(savedQuestion)
-
+  try {
+    const savedQuestion = await question.save()
+    return response.status(201).json(savedQuestion)
+  } catch (error) {
+    return response.json({
+      msg: `Something went wrong ${error}`
```

Comment:
`error` most likely is an object, so in this case the user will receive 

`Something went wrong [Object object]`

Also, when the server fails, the response status should be `500`

Reply:
You right, let me fix this with something I just learned yesterday